apps/docs/components/Navigation/Footer.tsx

```diff
@@ -37,18 +37,24 @@ const Footer = () => (
         ))}
       </ul>
     </div>
-    <hr className="border-scale-400 my-6"></hr>
+    <hr className="border-border my-6"></hr>
```

Comment:
can just be `border` if you want.

Reply:
I believe just `border` competes with actually setting all borders.